hypershift-operator/controllers/supportedversion/reconciler.go

```diff
@@ -22,8 +23,9 @@ import (
 )
 
 const (
-	configMapKey           = "supported-versions"
-	supportedVersionsLabel = "hypershift.openshift.io/supported-versions"
+	ConfigMapVersionsKey      = "supported-versions"
```

Comment:
This is resulting confusing at least for me, it'd be nice to include in the naming if we are referring to HOVersion or OCPVersions